Add a few additional items to the smart query

We need access to the source so we can figure out the complementary
install.

Signed-off-by: Mark Hatle <mark.hatle@windriver.com>

Index: smart-1.4.1/smart/commands/query.py
===================================================================
--- smart-1.4.1.orig/smart/commands/query.py
+++ smart-1.4.1/smart/commands/query.py
@@ -650,7 +650,10 @@ class TextOutput(NullOutput):
         if self.opts.show_format:
             info = pkg.loaders.keys()[0].getInfo(pkg)
             tags = dict(name=pkg.name, version=pkg.version,
-                        group=info.getGroup(), summary=info.getSummary())
+                        group=info.getGroup(), summary=info.getSummary(), 
+                        source=info.getSource(), 
+                        description=info.getDescription(),
+                        license=info.getLicense())
             fmt = self.opts.show_format.safe_substitute(tags)
             fmt = fmt.replace('\\t', "\t").replace('\\n', "\n")
             sys.stdout.write(fmt)
